front-end-tools/src/Main.tsx

```diff
@@ -89,7 +92,7 @@ export default function Main(props: ConnectionProps) {
         setModuleReference(target.value);
     };
 
-    const changeDropDownHandler = () => {
+    const changeInputParameterDropDownHandler = () => {
```

Comment:
Consider using `useCallback` to avoid redundant rerendering of the component(s) that use it.